### PR TITLE
Feature/gc institution

### PR DESCRIPTION
**Pull request by jmealing**
Create component for About the institution

To do:
[✓ ] French
[ ✓] update well-bold class to bold class (well-bold feature is modified in another PR) 
https://github.com/wet-boew/GCWeb/pull/2237
[ ] DTO guidance/content

DTO Design system PRs:
EN: https://github.com/canada-ca/design-system/pull/327
FR: https://github.com/canada-ca/systeme-conception/pull/217

### CODE REVIEW

File: components/gc-institution/gc-institution-en.html [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+language: en
+altLangPage: gc-institution-fr.html
+dateModified: 2023-08-09
+layout: no-container
+---
+<div class="container">
+	<h1>{{ page.title }}</h1>
+
+	<span class="wb-prettify all-pre hide"></span>
+	<p>TThe about the institution pattern is a section on an institutional landing page that provides links to corporate, program and policy information.</p>
```
Comment (delisma) [inline components/gc-institution/gc-institution-en.html:13]:
```suggestion
	<p>TThe about the institution pattern is a section on an institutional landing page that provides links to corporate, program and policy information.</p>
```

File: components/gc-institution/gc-institution-en.html [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+	<h2>On this page</h2>
+	<ul>
+		<!-- <li><a href="#inc-components">Included components</a></li> -->
+		<li><a href="#before-start">Before you start</a></li>
+		<li><a href="#accessibility-guidance">Accessibility guidance</a></li>
+		<li><a href="#variations">Variations</a></li>
+		<li><a href="#eval-report">Evaluation and report</a></li>
+		<li><a href="#technical-docs">Technical documentation</a></li>
+	</ul>
+
```
Comment (delisma) [inline components/gc-institution/gc-institution-en.html:24]:
```suggestion

```

File: components/gc-institution/gc-institution-en.html [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+		<li><a href="#before-start">Before you start</a></li>
+		<li><a href="#accessibility-guidance">Accessibility guidance</a></li>
+		<li><a href="#variations">Variations</a></li>
+		<li><a href="#eval-report">Evaluation and report</a></li>
+		<li><a href="#technical-docs">Technical documentation</a></li>
+	</ul>
+	
+	<h2 id="before-start">Before you start</h2>
+	<p>About the institution is a mandatory pattern on an institutional landing page.  It must include mandate and transparency links.</p>
+
```
Comment (delisma) [inline components/gc-institution/gc-institution-en.html:27]:
```suggestion

```

File: components/gc-institution/gc-institution-en.html [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+	<p>About the institution is a mandatory pattern on an institutional landing page.  It must include mandate and transparency links.</p>
+	
+	<ul>
+		<li><a href="">How to set up the institutional landing page</a></li>
+	</ul>
+
+	<h2 id="accessibility-guidance">Accessibility guidance</h2>
+
+	<p>List of accessibility guidances to be followed to ensure component is used in respect of the accessibility guideline and EN 301 549 standard (including WCAG 2.1 Level AA).</p>
+
```
Comment (delisma) [inline components/gc-institution/gc-institution-en.html:35]:
```suggestion

```

File: components/gc-institution/gc-institution-en.html [hunk 5]
```diff
@@ -0,0 +1,93 @@
+---
+title: About the institution pattern
+description: Documentation and working example for the about the institution pattern
+language: en
+altLangPage: gc-institution-fr.html
+dateModified: 2023-08-09
+layout: no-container
+---
+<div class="container">
+	<h1>{{ page.title }}</h1>
```
Comment (delisma) [inline components/gc-institution/gc-institution-en.html:10]:
```suggestion
	<h1 property="name" id="wb-cont" dir="ltr">{{ page.title }}</h1>
```

**Review by delisma — CHANGES_REQUESTED** [review]

**Comment by Garneauma** [comment]
@jmealing Could you please move this documentation in the newly created section /design-patterns ? Thank you.

**Comment by jmealing** [comment]
@Garneauma moved

File: design-patterns/gc-institution/gc-institution-en.html [hunk 6]
```diff
@@ -0,0 +1,90 @@
...
+		<li><a href="#variations">Variations</a></li>
+		<li><a href="#eval-report">Evaluation and report</a></li>
+		<li><a href="#technical-docs">Technical documentation</a></li>
+	</ul>
+
+	<h2 id="before-start">Before you start</h2>
+	<p>About the institution is a mandatory pattern on an institutional landing page. It must include mandate and transparency links.</p>
+
+	<ul>
+		<li><a href="">How to set up the institutional landing page</a></li>
```
Comment (Garneauma) [inline design-patterns/gc-institution/gc-institution-en.html:29]: Is this link intended to redirect to the design guidance?

**Review by Garneauma — CHANGES_REQUESTED** [review]

File: design-patterns/gc-institution/gc-institution-en.html [hunk 7]
```diff
@@ -0,0 +1,90 @@
...
+language: en
+altLangPage: gc-institution-fr.html
+dateModified: 2023-08-09
+layout: no-container
+---
+<div class="container">
+	<h1 property="name" id="wb-cont" dir="ltr">{{ page.title }}</h1>
+
+	<span class="wb-prettify all-pre hide"></span>
+	<p>TThe about the institution pattern is a section on an institutional landing page that provides links to corporate, program and policy information.</p>
```
[the remark this answers is not in the document]
Comment (jmealing) [reply design-patterns/gc-institution/gc-institution-en.html:13]: The style guide team says we are not capitalizing the "About". This is a choice made when the pattern is referenced in sentences

File: design-patterns/gc-institution/gc-institution-en.html [hunk 8]
```diff
@@ -0,0 +1,90 @@
...
+			<li><a href="#">Job opportunities</a></li>
+			<li><a href="#">Reports</a></li>
+			<li><a href="#">Compliance</a></li>
+			<li><a href="#">Enforcement notifications</a></li>
+			<li><a href="#">[Institutional link]</a></li>
+			<li>More: <a href="#">About the [Institution]</a></li>
+		</ul>
+	</section>
+
+	<div class="container">
```
[the remark this answers is not in the document]
Comment (jmealing) [reply design-patterns/gc-institution/gc-institution-en.html:61]: Was this fixed as part of accepting the previous changes, or do I still need to remove the container from somewhere?

[the remark this answers is not in the document]
Comment (Garneauma) [reply design-patterns/gc-institution/gc-institution-en.html:61]: You will need to remove this container.

File: design-patterns/gc-institution/gc-institution-en.html (hunk 6 above)
Comment (Garneauma) [reply design-patterns/gc-institution/gc-institution-en.html:29]: Ok, the link will need to be updated here once it is ready. I will keep this change request open until it is.

File: design-patterns/gc-institution/gc-institution-en.html (hunk 8 above)
[the remark this answers is not in the document]
Comment (jmealing) [reply design-patterns/gc-institution/gc-institution-en.html]: Done

File: design-patterns/gc-institution/gc-institution-en.html (hunk 6 above)
Comment (jmealing) [reply design-patterns/gc-institution/gc-institution-en.html]: Added the link, it's the old content, but it's going to the right place when we update everything together

**Comment by jmealing** [comment]
@Garneauma I'm going to work on the FR page now that EN seems to be all correct and reviewed

**Comment by Garneauma** [comment]
@jmealing As discussed, removed the utility classes `h3 mrgn-tp-md` to the the `<h2>`. The styles will be added to the ILP template itself.

File: design-patterns/gc-institution/index.json-ld [hunk 9]
```diff
@@ -0,0 +1,157 @@
...
+		"fr": "À propos de l’institution"
+	},
+	"description": {
+		"en": "Section on an institutional landing page that provides links to corporate, program and policy information",
+		"fr": "Section d’une page d’accueil institutionnelle offrant des liens vers des renseignements relatifs à un organisme, à un programme ou à une politique en particulier."
+	},
+	"modified": "2024-01-11",
+	"componentName": "gc-institution",
+	"status": "stable",
+	"version": "1.0",
```
Comment (duboisp) [inline design-patterns/gc-institution/index.json-ld:20]: We will need to use the documentation format that @Garneauma did for design pattern where there is no component stability indication neither a version number associated with it neither versioning api documentation.

File: design-patterns/gc-institution/gc-institution-fr.html [hunk 10]
```diff
@@ -0,0 +1,50 @@
...
+	],
+	"dateModified": "2024-03-07",
+	"pageclass": "wb-prettify all-pre"
+}
+---
+
+<p class="mrgn-bttm-lg">Vous trouverez ci-dessous un exemple pratique pour la configuration de conception « À propos de l'institution ».</p>
+
+<section class="row">
+	<div class="col-md-8 bold-content">
```
Comment (duboisp) [inline design-patterns/gc-institution/gc-institution-fr.html:18]: Why the `col-md-8`? That use seem to be a workaround of the limited width content feature.  We should use the limited width content feature instead.

A grid can be useful when there is multiple column, not just one which is use not for the column purpose but for limiting the content width.

May be we will need to extend the content limited width when there is a colcount of 2 feature.

**Review by duboisp — CHANGES_REQUESTED** [review]
A few change required
* Document as design pattern, where there is no versioning information.
* Remove the grid with only 1 column and leverage the limited width content instead.
* Extend the limited with content to also behave on list that leverage colcount-2 style

**Review by duboisp — CHANGES_REQUESTED** [review]
Review and tested, everything do work as expected. There is just one little change which is to move the alert fragment into a different folder. After that, this PR will be ready.

The only thing to fix are going to fix the potential merge conflict regarding the documentation_pattern.html file because the same file is present in the gc-intro design pattern PR.

File: _includes/design-patterns/gc-institution/gc-institution-alert.html
Comment (duboisp) [inline _includes/design-patterns/gc-institution/gc-institution-alert.html:1]: This file should be removed from here.

**Review by duboisp — CHANGES_REQUESTED** [review]
All good, please remove the include

**Review by duboisp — APPROVED** [review]
Reviewed and tested locally, everything work as expected.

Great work on this.

**Comment by duboisp** [comment]
We will wait for the completion of all the request in https://github.com/orgs/wet-boew/projects/4/views/1 + approval from DTO and PP before to merge it.

**Review by Garneauma — APPROVED** [review]

**Comment by Garneauma** [comment]
Rectification: This is a MINOR change.

**Pull request merged**